libs/blocks/icon-block/icon-block.css

```diff
@@ -331,6 +338,14 @@
     max-width: 300px;
   }
 
+  .two-up .icon-block.bio.inline .foreground,
```

Comment:
you could use a single selector - `[class*=-up] .icon-block.bio.inline .foreground`